src/v2/BaseLoginRouter.js

```diff
@@ -42,13 +42,13 @@ function beaconIsAvailable (Beacon, settings) {
   return true;
 }
 
-function loadLanguage (appState, i18n, assetBaseUrl, assetRewrite) {
+function loadLanguage (appState, languageCode, i18n, assetBaseUrl, assetRewrite) {
```

Comment:
what is this change for ?

Reply:
I remove `languageCode` from `AppState` hence has to get it from `settings`.